doc/kb_sdk_local_test_module.md

```diff
@@ -56,29 +56,28 @@ RUN rm -rf tarball.tgz
 ```
 
 because the latter does not remove the previous bloating layers, despite the "rm" command.
-<!--
-    RUN git clone https://github.com/torognes/vsearch
-    WORKDIR vsearch
-    RUN ./configure 
-    RUN make
-    RUN make install
-    WORKDIR ../
 
-You will also need to add your KBase SDK module, and any necessary data, to the Dockerfile.  For example:
+Final Note: Docker will rebuild everything from the first detected change in a dockerfile but pull everything upstream of that from cache. This means that if you are pulling in external data using RUN and a command like `git clone` or `wget` changes in those sources will not automatically be reflected in a rebuilt docker image unless the docker file changes at or before that import
 
-    RUN mkdir -p /kb/module/test
-    WORKDIR test
-    RUN git clone https://github.com/dcchivian/kb_vsearch
-    RUN git clone https://github.com/dcchivian/kb_vsearch_test_data
-    WORKDIR ../
--->
+#### <A NAME="get-token"></A>5B. Get a developer token
 
-#### <A NAME="build-tests"></A>5B. Build tests of your methods
+If you have already been approved as a KBase developer. A token may be generated from your [kbase account profile](https://narrative.kbase.us/#auth2/account) under the Developer Tokens tab.
 
-Edit the local test config file (`test_local/test.cfg`) with a KBase user account name and password (note that this directory is in .gitignore so will not be copied to github.  You're safe!):
+**But you haven't approved my developer request and I _really_ want to test!**
```

Comment:
Yeahhhhh.... this seems like a bad idea to me. It takes seconds for an Auth2 admin to find a user and add developer token status, so it seems to me the fix for this issue is to improve the speed at which we approve accounts.

Also, the catalog service should really get its developer list from the users that have developer token rights in the auth service, so that only one step is needed for approving a developer. That's later though.